cypress.config.ts

```diff
@@ -7,6 +7,8 @@ export default defineConfig({
     video: false,
     screenshotOnRunFailure: false,
     setupNodeEvents(on, config) {
+      config.defaultCommandTimeout = 20000
+      config.responseTimeout = 20000
```

Comment:
add a semicolon to the end of each line and return the config file:
```suggestion
      config.defaultCommandTimeout = 20000;
      config.responseTimeout = 20000;
      
      return config;
```